optaplanner-website-main/.gitignore

```diff
@@ -0,0 +1,16 @@
+# Local files
+/target
+/local
+
+# Eclipse, Netbeans and IntelliJ files
+/.*
+!.gitignore
+!.gitattributes
+!.htaccess
+/nbproject
+/*.ipr
+/*.iws
+/*.iml
+
+# Repository wide ignore mac DS_Store files
+.DS_Store
```

Comment:
Nitpick: Only the .gitignore in the root directory of a repo needs the ".DS_Store" ignore